drivers/dai/intel/dmic/dmic.c

```diff
@@ -785,7 +785,7 @@ static int dai_dmic_initialize_device(const struct device *dev)
 		.fifo =							\
 		{							\
 			.offset = DT_INST_REG_ADDR_BY_IDX(n, 0)		\
-				+ OUTDATA##n,				\
+				+ DT_INST_PROP_BY_IDX(n, fifo, 0),	\
```

Comment:
I think I agree with @andyross; this should be an int if you only expect one value inside. Unless there's something I'm missing? The commit logs don't have much information and there is no property description documenting what this is supposed to contain, so I'm going to -1 until we can clear this up.

If you switch to int, this just becomes `DT_INST_PROP(n, fifo)`.

Reply:
ok, should be like that now

Reply:
>  The commit logs don't have much information and there is no property description documenting what this is supposed to contain, so I'm going to -1 until we can clear this up.

This was a hint to improve the commit log and add a property description, just FYI. I'm going to dismiss my review now because it's moved to `int`, but I think these should be improved.